tests/radv/test_radv_ipv6_ra.py

```diff
@@ -24,10 +29,16 @@
 
 """
 
-@pytest.fixture(scope="module")
-def radv_test_setup(duthosts, rand_one_dut_hostname, ptfhost, tbinfo):
-    duthost = duthosts[rand_one_dut_hostname]
+@pytest.fixture(scope="module", autouse=True)
+def radv_test_setup(request, duthosts, ptfhost, tbinfo):
+    if 'dualtor' in tbinfo['topo']['name']:
+        # Select upper ToR to run RADV tests
+        duthost = duthosts[tbinfo['duts'][0]]
```

Comment:
You can just use `duthost = duthosts[0]`.

Reply:
Suggest make test_radv_router_advertisement and radv_test_setup consistently using either duthost or duthosts[0].

Reply:
using duthosts[0] now